packages/web/app/views/programRegistry/PatientProgramRegistryForm.jsx

```diff
@@ -38,17 +39,7 @@ export const PatientProgramRegistryForm = ({ onCancel, onSubmit, editedObject })
   const { data: program } = useQuery(['programRegistry', selectedProgramRegistryId], () =>
     selectedProgramRegistryId ? api.get(`programRegistry/${selectedProgramRegistryId}`) : null,
   );
-  const { data: { data: conditions } = {} } = useQuery(
```

Comment:
refactor to use the query util. The 2 different copies of the useQuery got out of sync.